packages/@lwc/integration-karma/test/synthetic-shadow/element-api/x/container/container.js

```diff
@@ -11,6 +11,6 @@ export default class Container extends LightningElement {
         createdDiv.appendChild(cmp);
 
         // this.template.insertBefore(createdDiv, lastParagraph);
-        templateDiv.insertBefore(createdDiv);
+        templateDiv.insertBefore(createdDiv, undefined);
```

Comment:
This was actually _already_ broken. We just didn't notice because the tests always eventually render an element in synthetic lifecycle mode (it happens for `Foo extends LightningElement {}` which doesn't go through the LWC compiler, which a few of our tests use), and Karma randomizes the order of tests, and we do 3 tests in CI until 1 passes, so this error was being hidden.

You can repro this error today in `master` by just making [this test](https://github.com/salesforce/lwc/blob/03686860484779367c4bcfac1cea288375c82b33/packages/%40lwc/integration-karma/test/synthetic-shadow/element-api/element-api.spec.js#L82) `fit()` rather than `it()`.

Reply:
I opened a separate issue to track this: #4319